front/migrations/20241129_fix_intercom_conversation_parents.ts

```diff
@@ -0,0 +1,99 @@
+import { CoreAPI } from "@dust-tt/types";
+import assert from "assert";
+import _ from "lodash";
+import { QueryTypes, Sequelize } from "sequelize";
+
+import config from "@app/lib/api/config";
+import { DataSourceModel } from "@app/lib/resources/storage/models/data_source";
+import { makeScript } from "@app/scripts/helpers";
+
+const { CORE_DATABASE_URI } = process.env;
+const SELECT_CHUNK_SIZE = 1000;
+const UPDATE_CHUNK_SIZE = 10;
+
+makeScript({}, async ({ execute }, logger) => {
+  assert(CORE_DATABASE_URI, "CORE_DATABASE_URI is required");
+
+  const coreSequelize = new Sequelize(CORE_DATABASE_URI as string, {
```

Comment:
You can get this through `getCorePrimaryDbConnection`.

Reply:
thanks, done!